base.js

```diff
@@ -263,6 +271,38 @@ const WIRE_ADAPTERS_WITH_RESTRICTED_USE = [
         module: 'lightning/uiRecordApi',
         identifier: 'getRecordUi',
     },
+    {
+        module: 'lightning/uiRelatedListApi',
+        identifier: 'getRelatedListsCount',
+    },
+    {
```

Comment:
Can you alphabetize this file to match the lightning modules so it's easier to track?